Merge project with doxygen and rest of calculating

floydsteam opened the pull request.

maxbachmann committed:
added some first ideas for the implementation

maxbachmann committed:
made some small style fixes like const

maxbachmann committed:
added HighPass Filter

added tests for the low pass filter
restructured fileorder

maxbachmann committed:
added + operator to build a bandpass

added bandpass

maxbachmann committed:
added operators

maxbachmann committed:
fixed + operator and splitted classes to hpp <-> cpp

maxbachmann committed:
changed getter/setter for components

maxbachmann committed:
added continous integration

maxbachmann committed:
added compiler version to travis

maxbachmann committed:
added compiler version to travis

maxbachmann committed:
added compiler version to travis

maxbachmann committed:
added compiler version to travis

maxbachmann committed:
added compiler version to travis

maxbachmann committed:
added compiler version to travis

maxbachmann committed:
added compiler version to travis

maxbachmann committed:
added compiler version to travis

maxbachmann committed:
added compiler version to travis

maxbachmann committed:
added compiler version to travis

maxbachmann committed:
added compiler version to travis

maxbachmann committed:
added compiler version to travis

maxbachmann committed:
Delete travis_gcc_cpp17

maxbachmann committed:
added more info to g++ output

maxbachmann committed:
made constructors explicit

maxbachmann committed:
gave m_value a default value

maxbachmann committed:
moved operators to passFilter bas class

floydsteam committed:
Started adding doxygen comments

floydsteam committed:
Merge branch 'master' into Development

maxbachmann committed:
fixed a forgotten merge conflict

maxbachmann commented:
Du hast in der Readme eine falsche Tabelle von Funktionen eingefügt

maxbachmann committed:
added virtual destructor to PassFilter

maxbachmann committed:
added end of file newline

removed a unneeded semicolon

maxbachmann committed:
improved settings for travis

maxbachmann committed:
changed to default value

maxbachmann committed:
constructors are now default

floydsteam committed:
Added doxygen to all class hpp files

floydsteam committed:
Finished with adding doxygen comments

maxbachmann commented:
@floydsteam Was hast du gemacht, dass dir den build bricht? 
Dachte du hast nur Kommentare hinzugefügt

floydsteam commented:
> 
> 
> @floydsteam Was hast du gemacht, dass dir den build bricht?
> Dachte du hast nur Kommentare hinzugefügt

Das frag ich mich auch grad, hab aber eigentlich wirklich nur Kommentare eingefügt...

floydsteam committed:
Added Bottom Frequenz to Bandpass

maxbachmann committed:
all High/LowPass are now PassFilters

floydsteam committed:
Added some calculation to Bandpass

maxbachmann committed:
fixed merge issues

maxbachmann committed:
Merge branch 'Development' of https://github.com/maxbachmann/passiveBandpass into Development

maxbachmann committed:
removed unused files

maxbachmann committed:
fixed some issues with the new structure

maxbachmann committed:
updated + operator

maxbachmann committed:
fixed a issue for Bandpass

until now you could create it directly which was a problem when passing 2 same PassFilters
now the Bandpass gets created by the PassFilter which is a friend

maxbachmann committed:
added combinedPassFilters

maxbachmann committed:
changed to combined Low/High pass

maxbachmann committed:
removed make_unique dependency for friend

maxbachmann committed:
changed operatos of components so they can calculate networks

added tests for the component classes to catch2

maxbachmann commented:
ich habe mal nochmal die Operatoren überarbeitet so dass jetzt mit R, C bzw L jeweils ein Netzwerk berechnet werden kann
und habe Tests für die component Klassen hinzugefügt

floydsteam committed:
Updated doxygen comments.

floydsteam committed:
Added forgotten doxygen comments and updated task description in Catch2.cpp and Readme

maxbachmann committed:
added more test case

maxbachmann committed:
Merge branch 'Development' of https://github.com/maxbachmann/passiveBandpass into Development

maxbachmann committed:
updated comment

maxbachmann committed:
added new Lines

maxbachmann committed:
removed new Line

maxbachmann committed:
updated travis

maxbachmann committed:
updated readme
